Classes/Cache/DatabaseCache.php

```diff
@@ -199,6 +199,15 @@ public function getUrlFromCacheBySpeakingUrl($rootPageId, $speakingUrl, $languag
 				'', 'expire'
 		);
 
+		// More than one urldata entry? Then prefer those with cHash
+		if (count($rows) > 1) {
+			if (preg_match('/cHash/', json_encode($rows))) {
+				$rows = array_filter($rows, function ($row) {
+					return strpos($row['original_url'], 'cHash') !== false ? true : false;
```

Comment:
I don't understand, I need one example of 2 original_url that will break with this code. Can you provide?